homeassistant/components/airthings_ble/__init__.py

```diff
@@ -42,6 +43,8 @@ async def _async_update_method():
         ble_device = bluetooth.async_ble_device_from_address(hass, address)
         airthings = AirthingsBluetoothDeviceData(_LOGGER, elevation, is_metric)
 
+        close_stale_connections(ble_device)
```

Comment:
Looks like this is missing an await?

Reply:
Yes... Tried different things, and forgot to push that. Thanks!